### PR TITLE
Dev/william/msquic 2.2

### PR DESCRIPTION
**Pull request by qzhuyan**

### CODE REVIEW

File: test/quicer_snb_SUITE.erl [hunk 1]
```diff
@@ -1701,16 +1701,20 @@ tc_conn_stop_notify_acceptor(Config) ->
...
                                                                        end, lists:seq(1, 100)),
                                                  {ok, Conn} = quicer:handshake(Conn),
-                                                 {error, closed} = quicer:accept_stream(Conn, []),
+                                                 case quicer:accept_stream(Conn, []) of
+                                                   {error, closed} -> ok;
+                                                   {ok, _Stream} -> ok
+                                                 end,
                                                  exit({normal, Acceptors})
                                              end),
+                 timer:sleep(1000),
```
Comment (qzhuyan) [inline test/quicer_snb_SUITE.erl:1717]: should remove...

File: test/quicer_SUITE.erl [hunk 2]
```diff
@@ -642,6 +642,9 @@ run_tc_conn_client_bad_cert(Config)->
         {error, stm_open_error, aborted} ->
           %% Depending on the timing, connection open could fail already.
           ok;
+        {error, stm_start_error, aborted} ->
+          %% Depending on the timing, connection open could fail already.
```
Comment (qzhuyan) [inline test/quicer_SUITE.erl:646]: s/open/start

File: CMakeLists.txt [hunk 3]
```diff
@@ -21,12 +21,23 @@ else()
...
+
 if (DEFINED ENV{QUIC_ENABLE_LOGGING})
   set(QUIC_ENABLE_LOGGING $ENV{QUIC_ENABLE_LOGGING})
+  set(QUIC_LOGGING_TYPE lttng)
 else()
   set(QUIC_ENABLE_LOGGING "OFF")
 endif()
 
+if (DEFINED ENV{QUIC_LOGGING_TYPE})
+  set(QUIC_ENABLE_LOGGING "ON")
```
Comment (paulozulato) [inline CMakeLists.txt:37]: `QUIC_LOGGING_TYPE` can activate `QUIC_ENABLE_LOGGING` even though this one is not set. And vice versa. Therefore, shouldn't be better to merge this two variables in a single one? Like `QUIC_LOGGING` which could be `off`, `lttng`, etc (any value different from `off` would be considered the logging type and would imply that logging is on)?

**Review by thalesmg — APPROVED** [review]

File: CMakeLists.txt (hunk 3 above)
Comment (qzhuyan) [reply CMakeLists.txt:37]: good idea but these are exposed by msquic and they are optional config,  I think it is just simple to keep them aligned with MsQuic.

**Review by paulozulato — APPROVED** [review]

**Review by ieQu1 — APPROVED** [review]

**Review by thalesmg — DISMISSED** [review]

**Review by zmstone — APPROVED** [review]

**Pull request merged**